test/index.test.js

```diff
@@ -240,32 +282,44 @@ test('test getSegments', async () => {
 test('test validateSegment', async () => {
   const url = 'https://analytics.adobe.io/api/test-company/segments/validate?rsid=111'
   const method = 'POST'
+  const api = 'validateSegment'
 
   mockResponseWithMethod(url, method, mock.data.validSegment)
   // check success response
   var res = await sdkClient.validateSegment(111, {})
-  expect(res.valid).toEqual(true)
+  expect(res.body.valid).toEqual(true)
```

Comment:
been leaving too many comments that are the same, but in general, I we shouldn't test for specific output here